include/librealsense2/h/rs_pipeline.h

```diff
@@ -41,6 +41,15 @@ extern "C" {
     */
     void rs2_pipeline_stop(rs2_pipeline* pipe, rs2_error ** error);
 
+    /**
+     * Set the device to be used in the pipline.
+     * The function is used to assign the device, ensuring the pipeline maintains the sensitivity values configured for
```

Comment:
This API is not for sensitivity, it's more generic than that right?
Sensitivity is 1 of the controls we can control
You can replace with
`useful when the user wish to set controls that cannot be set while streaming`
Sounds OK?

Reply:
Yes, changed